tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/js/dialog/extension-install-dialog.js

```diff
@@ -363,8 +256,52 @@ define(['require', 'lodash', 'jquery', 'log', 'backbone'],
                         self._extensionPartialModel.modal('show');
                     }
 
-                },
-            });
+                }
+
+        };
+
+        var constants = {
```

Comment:
move the constants to top or in the initialize method

Reply:
9ed0740 this commit will be resolving above request.